### PR TITLE
Getting Place details of business names from scraped datasets

### PR DESCRIPTION
**Pull request by casanted**
- Uses Place Search to find a place by the name from the scraped datasets 
- Uses Place Details to get more information about the place, including reviews, in order to assist in searching through these places

### CODE REVIEW

**Comment by graceehuu** [comment]
LGTM! I think you might've forgotten to request reviews

File: capstone/src/main/webapp/style.css [hunk 1]
```diff
@@ -24,28 +24,53 @@
...
   float: right;
-  font-size: 36px;
+  font-weight: bold;
+  margin-left: 5px;
 }
 
-.info {
-  display: none;
+.active::after {
+  content: "\2212";
```
Comment (graceehuu) [inline capstone/src/main/webapp/style.css:65]: what does this do?

Comment (casanted) [reply capstone/src/main/webapp/style.css:65]: That makes the + sign in the name button when the expanded view is not open, and the - sign when it is opened, in the results list

**Comment by casanted** [comment]
> LGTM! I think you might've forgotten to request reviews

I added it as a field in the details request, so it should be taken care of.

**Comment by terencedignon** [comment]
Hey Carolyn, why is this deleting thanks.html? Is it a rebase issue?

**Comment by casanted** [comment]
> Hey Carolyn, why is this deleting thanks.html? Is it a rebase issue?

Oh we decided to get rid of the thank you page, and just have the user be redirected to the map (main.html) after they submit their feedback.

**Pull request merged**